vnext/Microsoft.ReactNative/Pch/pch.h

```diff
@@ -53,3 +53,7 @@
 
 #include <activeObject/activeObject.h>
 #include <future/future.h>
+
+// https://github.com/microsoft/microsoft-ui-xaml/issues/3133
+#include <winrt/Microsoft.UI.Xaml.Controls.h>
```

Comment:
Does this need to be changed to wrt WinUI3?

Reply:
@jonthysell good question, it does not :) Bug 3133 is that if you reference winui 2.x then xaml compiler will try to generate type info in your app for any type marked Bindable which includes TreeViewNode, even if you never use TreeViewNode.
The same header exists in 3.0 so no change necessary.